.all-contributorsrc

```diff
@@ -36,6 +36,315 @@
         "code",
         "design"
       ]
+    },
+    {
+      "login": "martinratinaud",
+      "name": "Martin Ratinaud",
+      "avatar_url": "https://avatars.githubusercontent.com/u/4191809?v=4",
+      "profile": "https://github.com/martinratinaud",
+      "contributions": [
+        "code",
+        "infra"
+      ]
+    },
+    {
+      "login": "Cli4d",
+      "name": "Clifford Ouma",
+      "avatar_url": "https://avatars.githubusercontent.com/u/56266330?v=4",
+      "profile": "https://github.com/Cli4d",
+      "contributions": [
+        "review",
+        "doc"
+      ]
+    },
+    {
+      "login": "emmanuellar",
+      "name": "Amarachi Johnson-Ubah",
+      "avatar_url": "https://avatars.githubusercontent.com/u/41474439?v=4",
+      "profile": "https://github.com/emmanuellar",
+      "contributions": [
+        "review",
+        "doc"
+      ]
+    },
```

Comment:
I'm confused: is it normal that “staff” members are also duplicated as “contributors”? 🤔 I'm not against it, this could be relevant, but I had not understood this this way and I want to double-check it is voluntary.

Reply:
It's voluntary.  As the list of contributors is also displayed in the repository, it seemed logical to me to display all the contributors.

Reply:
I understand. I want to challenge the display in the repository, as it seems misleading to me: those are not contributors to the repository (which is the default usage of All Contributors) but to the overall product. I thus believe we should not make it visible on the repository itself, only on the website.